### PR TITLE
add install-mysql-database-server-on-ubuntu [EN]

### PR DESCRIPTION
**Pull request by DerGonzi**
I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: David Joswig <info@derjoswig.de>

### CODE REVIEW

File: tutorials/install-mysql-database-server-on-ubuntu/01.en.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-mysql-database-server-on-ubuntu"
+slug: "install-mysql-database-server-on-ubuntu"
+date: "2021-03-25"
+title: "Install a MySQL server on Ubuntu"
+short_description: "This is a tutorial which teaches you how to install a MySQL database server."
+tags: ["MySQL", "Database"]
+author: "David Joswig"
+author_link: "https://github.com/dergonzi"
+author_img: "https://avatars.githubusercontent.com/u/35921566?s=60&v=4"
```
Comment (rook59) [inline tutorials/install-mysql-database-server-on-ubuntu/01.en.md:11]:
```suggestion
author_img: "https://avatars.githubusercontent.com/u/35921566"
```

File: tutorials/install-mysql-database-server-on-ubuntu/01.en.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+## Introduction
+
+In this tutorial is about the installation and the initial configuration of a MySQL server.
+
+**Prerequisites**
+
+- A [Hetzner Cloud](https://www.hetzner.com/cloud) account
+- A Hetzner Cloud project for your Ubuntu Server
+- A Hetzner Cloud Ubuntu Server
```
Comment (rook59) [inline tutorials/install-mysql-database-server-on-ubuntu/01.en.md:26]: You just need an Ubuntu installation, doesn't matter from which provider it is ;)

File: tutorials/install-mysql-database-server-on-ubuntu/01.en.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+UPDATE mysql.user SET host='%' WHERE user='root' AND host='localhost';
+```
+
+To apply these change you have to restart the mysql server: `sudo service mysql restart`.
+
+#### Now you can connect from anywhere, remember to secure your server properly and that this is **NOT recommended** for production use.
+
+## Conclusion
+
+Now you can install and configure MySQL server.
```
Comment (rook59) [inline tutorials/install-mysql-database-server-on-ubuntu/01.en.md:250]: Well, the MySQL server is now installed after following this tutorial

File: tutorials/install-mysql-database-server-on-ubuntu/01.en.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+
+_Type **Y**, then press **enter**_
+
+```Shell
+Success.
+
+All done!
+```
+
+# Step 2 - Edit config file to enable remote access
```
Comment (rook59) [inline tutorials/install-mysql-database-server-on-ubuntu/01.en.md:158]: This should be optional, too. I guess in most cases you install MySQL next to your application on the same server. Maybe add some further information when you need to enable remote access (e.g. if you have different servers for the application and the database)

**Review by rook59 — COMMENTED** [review]
Thank you for your submission. Please take a look at my notes

**Comment by DerGonzi** [comment]
@rook59 changed

**Pull request merged**